exchanges/binanceus/binanceus_types.go

```diff
@@ -0,0 +1,1180 @@
+package binanceus
+
+import (
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/shopspring/decimal"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/trade"
+)
+
+var (
+	// BinanceRequestParamsOrderLimit Limit order
+	BinanceRequestParamsOrderLimit = RequestParamsOrderType("LIMIT")
+	// BinanceRequestParamsOrderMarket Market order
+	BinanceRequestParamsOrderMarket = RequestParamsOrderType("MARKET")
+	// BinanceRequestParamsOrderStopLoss STOP_LOSS
+	BinanceRequestParamsOrderStopLoss = RequestParamsOrderType("STOP_LOSS")
+	// BinanceRequestParamsOrderStopLossLimit STOP_LOSS_LIMIT
+	BinanceRequestParamsOrderStopLossLimit = RequestParamsOrderType("STOP_LOSS_LIMIT")
+	// BinanceRequestParamsOrderTakeProfit TAKE_PROFIT
+	BinanceRequestParamsOrderTakeProfit = RequestParamsOrderType("TAKE_PROFIT")
+	// BinanceRequestParamsOrderTakeProfitLimit TAKE_PROFIT_LIMIT
+	BinanceRequestParamsOrderTakeProfitLimit = RequestParamsOrderType("TAKE_PROFIT_LIMIT")
+	// BinanceRequestParamsOrderLimitMarker LIMIT_MAKER
+	BinanceRequestParamsOrderLimitMarker = RequestParamsOrderType("LIMIT_MAKER")
+)
+
+const wsRateLimitMilliseconds = 250
```

Comment:
I understand that this is a generous rate limit if we look at the API docs here: https://docs.binance.us/#websocket-rate-limits where it could be as low as 200...

However, I'm receiving this:
```
[WARN]  | WEBSOCKET | 11/08/2022 16:22:29 | BinanceUS has 1024 subscription limit, only subscribing within limit. Requested 1088
[WARN]  | WEBSOCKET | 11/08/2022 16:22:34 | Binanceus websocket has been disconnected. Reason: websocket: close 1008 (policy violation): Too many requests
[ERROR] | WEBSOCKET | 11/08/2022 16:22:34 | Binanceus subscription failure: Binanceus websocket connection: cannot send message to a disconnected websocket
```

Please change this to 
```suggestion
const wsRateLimitMilliseconds = 300
```
Given the lack of auth websocket functions and that its only during subscriptions + ping pong messaging, this should have very little impact